tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserSBytePerfTests.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+
+namespace System.Text.Primitives.Tests
+{
+    public partial class PrimitiveParserSBytePerfTests
+    {
+        private const int InnerCount = 100000;
+
+        private static readonly string[] s_SByteTextArray = new string[17]
+        {
+           "95",
+            "2",
+            "112",
+            "-112",
+            "-21",
+            "-2",
+            "114",
+            "-114",
+            "-124",
+            "117",
+            "-117",
+            "-14",
+            "14",
+            "74",
+            "21",
+            "83",
+            "-127"
+        };
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("107")] // standard parse
+        [InlineData("127")] // max value
+        [InlineData("0")]
+        [InlineData("-128")] // min value
+        [InlineData("147")]
+        [InlineData("2")]
+        [InlineData("105")]
+        [InlineData("-111")]
+        [InlineData("-21")]
+        [InlineData("-2")]
+        [InlineData("-13")]
+        [InlineData("-8")]
+        [InlineData("-83")]
+        [InlineData("+127")]
+        [InlineData("+21")]
+        [InlineData("+2")]
+        [InlineData("00000000000000000000123abcdfg")]
+        [InlineData("2abcdefghijklmnop")]
+        [InlineData("14abcdefghijklmnop")]
+        [InlineData("-14abcdefghijklmnop")]
+        [InlineData("-21abcdefghijklmnop")]
+        [InlineData("-2abcdefghijklmnop")]
+        [InlineData("+14abcdefghijklmnop")]
+        [InlineData("+21abcdefghijklmnop")]
+        [InlineData("+2abcdefghijklmnop")]
+        [InlineData("+111abcdefghijklmnop")]
+        [InlineData("+000000000000000000123abcdfg")]
+        private static void PrimitiveParserByteSpanToSByte_BytesConsumed(string text)
+        {
+            byte[] utf8ByteArray = Text.Encoding.UTF8.GetBytes(text);
+            var utf8ByteSpan = new ReadOnlySpan<byte>(utf8ByteArray);
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        PrimitiveParser.InvariantUtf8.TryParseSByte(utf8ByteSpan, out sbyte value, out int bytesConsumed);
+                        TestHelper.DoNotIgnore(value, bytesConsumed);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        private static void PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength()
```

Comment:
Can you also add PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength_Baseline, similar to the following?
https://github.com/dotnet/corefxlab/blob/master/tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserInt32PerfTests.cs#L270